tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -825,7 +826,7 @@ def _post_fit(self, keep_only_best=False, refit_full=False, set_best_to_refit_fu
             self.delete_models(models_to_keep='best', dry_run=False)
 
         if calibrate:
-            if self.problem_type in PROBLEM_TYPES_CLASSIFICATION:
+            if self.problem_type in PROBLEM_TYPES_CLASSIFICATION + [QUANTILE]:
                 self._calibrate_model()
             else:
                 logger.log(30, 'WARNING: calibrate is only applicable to classification problems')
```

Comment:
should add quantile-regression to this statement

Reply:
I'd put the current CONFORMALIZATION into `_calibrate_model`.
We could separately define 1)calibrate an 2) conformal, but for simplicity I treat them together.

Reply:
I think it's fine as it currently is